packages/libqtxdg/build.sh

```diff
@@ -0,0 +1,11 @@
+TERMUX_PKG_HOMEPAGE=https://lxqt.github.io
+TERMUX_PKG_DESCRIPTION="Qt 5 implementation of freedesktop.org XDG specifications"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="Simeon Huang <symeon@librehat.com>"
+TERMUX_PKG_VERSION=3.7.1
+TERMUX_PKG_SRCURL="https://github.com/lxqt/libqtxdg/releases/download/${TERMUX_PKG_VERSION}/libqtxdg-${TERMUX_PKG_VERSION}.tar.xz"
+TERMUX_PKG_SHA256=477cbe76b3305071ff5f5bfa31dbcddcc51f3434b9ed75d91988219296d88a9b
+TERMUX_PKG_DEPENDS="qt5-qtbase"
+TERMUX_PKG_BUILD_DEPENDS="cmake, lxqt-build-tools, glib, qt5-qtsvg, qt5-qttools-cross-tools"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DCMAKE_INSTALL_PREFIX=${TERMUX_PREFIX}"
```

Comment:
Is there a reason for specifying CMAKE_INSTALL_PREFIX manually?

This is already specified in https://github.com/termux/termux-packages/blob/master/scripts/build/configure/termux_step_configure_cmake.sh and shouldn't be specified manually unless you are overriding the whole CMake configuration step.